index.html

```diff
@@ -0,0 +1,430 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width" />
+      <meta name="viewport" content="initial-scale=1">
+      <title>Advanced Presentation</title>
+      <link rel='stylesheet' href='css/stylesheet.css' type='text/css'>
+      <link href='https://fonts.googleapis.com/css?family=Droid+Sans' rel='stylesheet' type='text/css'>
+      <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Lato:400,100,300' type='text/css'>
+    </head>
+    <body>
+
+      <header>
+<!-- Menu Bar -->
+        <nav class="mobile-menu">
+          <label for="show-menu" class="show-menu">
+             <span class="icon-menu"></span>
+          </label>
+         <input type="checkbox" id="show-menu">
+          <ul class="menu">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="index.html#services">Services</a></li>
+            <li><a href="index.html#portfolio">Portfolio</a></li>
+            <li><a href="index.html#about">About</a></li>
+            <li><a href="index.html#team">Team</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- Hi There Avengers first section -->
+      <section class="p-home">
+        <div class="l-home-spacer"> </div>
```

Comment:
Not a huge fan of placing DOM just for presentation. This is meaningless to the document. In the future, I would look to hinge this presentation on other needed DOM.